fiat-roles/src/main/java/com/netflix/spinnaker/fiat/config/RedisConfig.java

```diff
@@ -36,16 +35,28 @@ public GenericObjectPoolConfig redisPoolConfig() {
   @Bean
   public JedisPool jedisPool(@Value("${redis.connection:redis://localhost:6379}") String connection,
                              @Value("${redis.timeout:2000}") int timeout,
+                             Registry registry,
                              GenericObjectPoolConfig redisPoolConfig) {
-    return createPool(redisPoolConfig, connection, timeout);
+    final JedisPool jedisPool = createPool(registry, redisPoolConfig, connection, timeout);
+    if (jedisPool instanceof InstrumentedJedisPool) {
+      final GenericObjectPool pool = ((InstrumentedJedisPool) jedisPool).getInternalPoolReference();
```

Comment:
This call is failing. Going to rethink how to get a ref to the internal pool. Likely will need a small kork change.